features/analytics.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Analytics
+  In order to understand usage to improve our services
+  As a developer
+  I want to be able to track user behaviour
+
+  @javascript
+  Scenario: Ecommerce tracking
```

Comment:
2. the only eCommerce tracking i've got here just checks for the existance of the tag... in a better world i'd trigger a GA event and check the payload matches expectations. But that's a bit of javascript testing I'd need to swot up on...

Not sure if it's worth it? or if we just trust that if the tags exist... then ga will pick up on them correctly.